src/Select/SelectOverlay.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC, useMemo } from 'react';
+import { SelectProps, SelectOption } from './Select';
+import Select, { ValueType } from 'react-select';
+import { SELECT_OVERLAY_STYLES } from './selectStyles';
+import theme from 'src/styles/theme';
+const SelectOverlay: FC<SelectProps> = ({ isSearchable = false, disabled, id, options, onChange, value, ...rest }) => {
+    const { currentValue, defaultValue, selectId } = useMemo(
+        () => ({
+            currentValue: options.find((option) => option.value === value),
+            defaultValue: options[0],
+            selectId: id ? `select-overlay-${id}` : undefined
+        }),
+        [options, value]
+    );
+
+    const handleChange = (option: any) => {
```

Comment:
I'm not super familiar with react-select, but can the `option: any` be typed like `option: SelectOption`?

Reply:
I have tried to do the same but somehow its not working for the latest react-select version.